src/test/java/core/basesyntax/FruitTest.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+import core.basesyntax.model.Fruit;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class FruitTest {

```

Comment:
U don`t need tests for models, only for class which represent some logic, like service